slownik.rst

```diff
@@ -0,0 +1,178 @@
+
+Funkcja
+  Zadanie, które ma spełnić system informatyczny.
+
+FW
+  ang. Firewall (zapora ochronna).
+
+GitHub
+  Serwis społecznościowy, w którym można udostępnić kod źródłowy napisanego przez siebie programu lub biblioteki programistycznej, śledzić różnice pomiędzy poszczególnymi wersjami oraz zgłaszać problemy z umieszczonym tam oprogramowaniem.
+
+IAM
+  ang. Identity and Access Management — usługa pozwalająca na zarządzanie tożsamością, uwierzytelnianie i zarządzanie uprawnieniami/rolami użytkowników.
+
+IDS
+  ang. Intrusion Detection System — urządzenia poprawiające bezpieczeństwo sieci komputerowych poprzez wykrywanie ataków w czasie rzeczywistym.
+
+Implementacja
+  [za PWN] wersja programu, języka programowania lub systemu operacyjnego przystosowana do pracy na określonym typie komputera; też: przystosowanie programu, języka programowania lub systemu operacyjnego do typu komputera.
+
+Instancja
+  Maszyna lub maszyny wirtualne, w ramach których działa instalacja platformy specyficzna dla konkretnego urzędu. Przydzielona dla jednej instytucji kopia Platformy, na której będzie uruchamiany pewien zbiór Aplikacji.
+
+IPS
+  ang. Intrusion Prevention System – urządzenia sieciowe poprawiające bezpieczeństwo sieci komputerowych przez wykrywanie i blokowanie ataków w czasie rzeczywistym.
+
+JST
+  Jednostka bądź jednostki samorządu terytorialnego: gmina, powiat, województwo.
+
+MVP
+  ang. Minimum Viable Product - podstawowa wersja systemu zapewniająca podstawową funkcjonalność umożliwiającą wprowadzenie na rynek.
+
+Marketplace
+  Marketplace będzie miejscem, gdzie przedstawiciel instytucji może zarejestrować się i utworzyć swoją instancję platformy aplikacje.gov.pl. Ten komponent będzie też odpowiadał za instalowanie poszczególnych aplikacji oraz komunikację z chmurą obliczeniową w celu np. skonfigurowania nowej instancji lub zmiany parametrów istniejącej instancji.
+
+Metadane
+  Zestaw usystematyzowanych informacji, logicznie powiązanych z przesyłką, sprawą lub inną dokumentacją, ułatwiających ich wyszukiwanie, kontrolę, zrozumienie i długotrwałe przechowanie oraz zarządzanie
+
+NASK
+  NASK jest instytutem badawczym prowadzącym działalność badawczo-rozwojową w zakresie opracowywania rozwiązań zwiększających efektywność, niezawodność i bezpieczeństwo sieci teleinformatycznych oraz innych złożonych systemów sieciowych oraz działalność edukacyjną, w tym w zakresie popularyzacji idei społeczeństwa informacyjnego
+
+Kod
+  Treść programu komputerowego.
+
+Komponent
+  Część składowa systemu realizująca konkretny zbiór funkcji.
+
+KPI (kluczowy wskaźnik efektywności)
+  Finansowy i niefinansowy wskaźnik pomiaru stopnia realizacji celów. Powinien być wyrażony w liczbach, procentach itd.
+
+Keyless Signatures’ Infrastructure
+  Usługa znakowania czasem w technologii linked timestamping, działająca zgodnie z modelem przedstawionym w dokumencie `„Keyless Signatures’ Infrastructure: How to Build Global Distributed Hash-Trees”`_.
```

Comment:
Zastanawiam się, na ile słownik ma być zrozumiały dla mniej technicznych osób? Przydałoby się wyjaśnić, co to jest `linked timestamping`.